docker/Dockerfile_local

```diff
@@ -21,9 +18,18 @@ RUN apt-get update -qq && \
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+COPY docker/docker-entrypoint.sh /usr/local/bin/
+COPY ./Gemfile* ./
```

Comment:
What is your hope of specifying the Gemfile like this? We explicitly exclude the lock file in .dockerignore.

Reply:
Missed that, I used a line I had from another file.